test/cmd/legacy-script.sh

```diff
@@ -484,6 +485,15 @@ runTests() {
     record_command run_kubectl_old_print_tests
   fi
 
+  ################
+  # Kubectl exec #
+  ################
+
+  if kube::test::if_supports_resource "${pods}" ; then
+    record_command run_kubectl_exec_pod_tests
+    record_command run_kubectl_exec_resource_name_tests
```

Comment:
Not quite, in this test you require pods, deployments, replicaset and configmaps too.